src/components/NewChapterForm.tsx

```diff
@@ -0,0 +1,269 @@
+import React from "react";
+import { z } from "Zod";
+import { useForm, SubmitHandler } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { ChapterRequest } from "src/app/api/chapter-request/route.schema";
+import { ErrorMessage } from "@hookform/error-message";
+
+const NewChapterForm = () => {
+  const title = [
+    "First Name",
+    "Last Name",
+    "University Email",
+    "Phone Number",
+    "University",
+    "University Address",
+  ];
+
+  // const schema = ChapterRequest
+  // .object({
+  //   firstName: ChapterRequest.string().required(),
+  //   lastName: ChapterRequest.string().required(),
+  //   universityEmail: ChapterRequest.string().email("This is not a valid email").required(),
+  //   phoneNumber: ChapterRequest.string().length(10, "Phone number must be 10 digits").required(),
+  //   university: ChapterRequest.string().required(),
+  //   universityAddress: ChapterRequest.string().required(),
+  //   leadershipExperience: ChapterRequest.string().required(),
+  //   motivation: ChapterRequest.string().required(),
+  //   availabilities: z.string(),
+  //   questions: z.string(),
+  // })
+  // .required()
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<ValidationSchema>({
+    resolver: zodResolver(ChapterRequest),
+  });
+
+  const onSubmit: SubmitHandler<typeof ChapterRequest> = (data) =>
+    console.log(data);
+
+  // const validationSchema = z
+  // .object({
+  //   firstName: z.string().min(1, { message: "Firstname is required" }),
+  //   lastName: z.string().min(1, { message: "Lastname is required" }),
+  //   email: z.string().min(1, { message: "Email is required" }).email({
+  //     message: "Must be a valid email",
+  //   })
+  // })
+
+  type ValidationSchema = z.infer<typeof ChapterRequest>;
+
+  return (
+    <div className="grid place-items-center px-10 py-5">
+      <div className="h-fit w-11/12  rounded-md bg-dark-teal px-9 py-10 text-lg text-white">
+        <form onSubmit={handleSubmit(onSubmit)}>
+          <div className="grid grid-cols-2 gap-x-10 gap-y-5">
+            <div className="w-full ">
+              <div>First Name</div>
+              <input
```

Comment:
Should we add extra padding for `<input .../>?

![image](https://github.com/JumboCode/TheLegacyProject2/assets/74647679/61cf23a3-5825-4e9d-a291-7c73cd8d36ca)